packages/block-editor/src/components/block-inspector/index.js

```diff
@@ -66,7 +66,7 @@ const BlockInspector = ( { showNoBlockSelectedMessage = true } ) => {
 		return (
 			<div className="block-editor-block-inspector">
 				<MultiSelectionInspector />
-				<InspectorControls.Slot />
+				<InspectorControls.Slot bubblesVirtually={ false } />
```

Comment:
I'm not sure this is the right fix, If I remember properly, this breaks context propagation in the fills. For instance if there's a context being provided around the slot like we do sometimes for `BlockEditContextProvider`, the inspector controls won't receive these values anymore. I remember in the past the impossibility to use RichText inside of Inspector controls because of this.

For these reasons, we've preferred moving all slots to `bubblesVirtually`, The other implementation might be here just for native usage (where portals are not supported)

Reply:
> I'm not sure this is the right fix, If I remember properly, this breaks context propagation in the fills. 

Yeah, it was more of a 🤞 so thanks for the warning.

The fallback, if we can't come up with a neater way, would be to try to stop propagation on key events for these particular text controls. Maybe even targeting `del` and `backspace` specifically. 🤷

Reply:
> be to try to stop propagation on key events for these particular text controls. Maybe even targeting del and backspace specifically

What the heck. 😄 

https://github.com/WordPress/gutenberg/pull/37472

It's more low level, but might be a backup approach if we can't target the events at a higher level.